renku/service/controllers/api/mixins.py

```diff
@@ -123,7 +138,7 @@ def sync(self, remote="origin"):
         if self.project_path is None:
             raise RenkuException("unable to sync with remote since no operation has been executed")
 
-        _, remote_branch = repo_sync(Repo(self.project_path), remote=remote)
+        _, remote_branch = repo_sync(git.Repo(self.project_path), remote=remote)
```

Comment:
Could we keep the old notation? Why are we changing it here?

Reply:
Oh good point. I think this was leftover code from debugging something